web/styles/recent_topics.css

```diff
@@ -25,6 +25,14 @@
          */
         min-height: 100vw;
 
+        & table {
+            table-layout: fixed;
```

Comment:
Changing to fixed table-layout can cause ugly overflow problems if any of the table entries are long; did you investigate that?

Reply:
I tried but couldn't see any such overflow! I tried having the stream name and topic name both at max length.